pkg/skaffold/runner/v1/dev.go

```diff
@@ -226,6 +228,22 @@ func (r *SkaffoldRunner) Dev(ctx context.Context, out io.Writer, artifacts []*la
 			continue
 		}
 
+		var absMatches []string
+		// Add absolute typed rules
+		if artifact.Sync != nil {
+			for _, pt := range artifact.Sync.Manual {
+				if filepath.IsAbs(pt.Src) {
+					// list files and add to dependencies list
+					matches, err := doublestar.Glob(pt.Src)
+					if err != nil {
+						return fmt.Errorf("pattern error for %q: %w", pt.Src, err)
```

Comment:
Please add actionable error message. eg. https://github.com/GoogleContainerTools/skaffold/blob/master/DEVELOPMENT.md#adding-actionable-error-messages-to-code

This Error happens in "INIT" Phase so adding error code like:
"INIT_SYNC_ABS_PATH_ERR" would be great!
Please add these in enums.proto file. I will update the instruction in Development.md from "skaffold.proto" to enums.proto